packages/statemanager/src/stateManager.ts

```diff
@@ -333,20 +570,20 @@ export class DefaultStateManager extends BaseStateManager implements StateManage
    */
   async getProof(address: Address, storageSlots: Uint8Array[] = []): Promise<Proof> {
     const account = await this.getAccount(address)
+    if (!account) {
+      throw new Error(`getProof() can only be called for an existing account`)
```

Comment:
I disagree, you can create a proof that an account does not exist in the trie. (non-existance proof)

Reply:
Ah, this non-existance thing 😋, thanks again @acolytec3 for updating! 🔥 🙏

(I am generally still no proof expert, so the proof parts are the weakest parts of this PR)